app/views/devise/registrations/new.html.erb

```diff
@@ -1,25 +1,33 @@
-<h2>Sign up</h2>
+<div class="container">
+  <div class="row justify-content-center my-3">
+    <div class="col-12 col-lg-6">
+      <h2 class="py-2">Sign up</h2>
 
-<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-  <%= f.error_notification %>
+      <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+        <%= f.error_notification %>
 
-  <div class="form-inputs">
-    <%= f.input :email,
-                required: true,
-                autofocus: true,
-                input_html: { autocomplete: "email" }%>
-    <%= f.input :password,
-                required: true,
-                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
-                input_html: { autocomplete: "new-password" } %>
-    <%= f.input :password_confirmation,
-                required: true,
-                input_html: { autocomplete: "new-password" } %>
-  </div>
+        <div class="form-inputs">
+          <%= f.input :username, autofocus: true %>
+          <%= f.input :email,
+                      required: true,
+                      autofocus: true,
+                      input_html: { autocomplete: "email" }%>
+          <%= f.input :address, autofocus: true %>
```

Comment:
Not sure if having 2 autofocus will cause an issue, I'm unfamiliar with the code.

Reply:
Checked in http://localhost:3000/users/sign_up and it was ok!